src/pages/index.en.tsx

```diff
@@ -259,9 +260,9 @@ export const LandingPageScaffolding = (props: ContentfulContent) => (
           <div className="column is-6 is-12-mobile is-flex is-flex-direction-column">
             <h1 className="mb-6">{props.content.partnershipsSectionTitle}</h1>
             <div className="has-background-success p-8 pt-11 is-flex-grow-1 is-flex is-flex-direction-column">
-              <h2 className="mb-11">
+              <ResponsiveElement desktop="h2" touch="h3" className="mb-11">
```

Comment:
oh this responsiveelement is also very cool haha

Reply:
Thanks! Yeah it's been helpful. I'd like to improve it to be more flexible with screen size options